core/src/Cuda/Kokkos_Cuda_Instance.hpp

```diff
@@ -487,16 +487,16 @@ class CudaInternal {
 #if (defined(KOKKOS_ENABLE_IMPL_CUDA_MALLOC_ASYNC) && CUDART_VERSION >= 11020)
   template <bool setCudaDevice = true>
   cudaError_t cuda_malloc_async_wrapper(void** devPtr, size_t size,
-                                        cudaStream_t hStream == nullptr) const {
+                                        cudaStream_t hStream = nullptr) const {
     if constexpr (setCudaDevice) set_cuda_device();
-    return cudaMallocAsync(devPtr, size, get_input_stream(stream));
+    return cudaMallocAsync(devPtr, size, get_input_stream(hStream));
```

Comment:
what did the "H" stand for?

Reply:
I used `hStream` to match the cuda API, but looking at the documentation (this is in the ["Stream Ordered Memory Allocator" module](https://docs.nvidia.com/cuda/cuda-runtime-api/group__CUDART__MEMORY__POOLS.html#group__CUDART__MEMORY__POOLS)) it's not obvious to me exactly why they use both `stream` and `hStream` there, especially since the only function which uses `stream` refers to the input as `hStream` in the description. Maybe this is a typo?

But looking in the ["Stream Management" module](https://docs.nvidia.com/cuda/cuda-runtime-api/group__CUDART__STREAM.html#group__CUDART__STREAM), `hStream` inputs are described as "Handle to the stream to be queried" where `stream` inputs are described "The stream to query".